terraform/environment/.envrc

```diff
@@ -1,5 +1,4 @@
 export TF_VAR_default_role=operator
 export TF_VAR_management_role=operator
```

Comment:
you'll probably find that management_role can be dropped and just replaced by default_role now